genkit-tools/common/src/eval/localFileDatasetStore.ts

```diff
@@ -32,6 +32,11 @@ import { logger } from '../utils/logger';
  * A local, file-based DatasetStore implementation.
  */
 export class LocalFileDatasetStore implements DatasetStore {
+  readonly SHORT_UUID_LENGTH = 6;
+  readonly ID_GEN_MAX_ATTEMPTS = 5;
+  readonly DISPLAY_NAME_MAX_LENGTH = 36;
+  readonly DATASET_ID_MAX_LENGTH = 18;
```

Comment:
UUIDv4 generates 36 character IDs. Can we keep this also 36? And maybe increase the SHORT_UUID_LENGTH to 18?